@kiva/kv-components/src/vue/KvSecondaryNav.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+	<kv-theme-provider
+		:theme="themeStyle"
+		class="kv-tailwind"
+	>
+		<div
+			class="tw-z-1 tw-w-full"
+			:class="isBelowElement ? 'tw-fixed' : 'tw-absolute'"
```

Comment:
It seems that when scrolling starts and the fixed class is applied the component moves a little to the right. It also breaks out of the containing element.